lib/state/appState.js

```diff
@@ -68,6 +68,7 @@ appState.getInitial = function() {
 
   return {
     dropMenu: false,
+    howToUpdateKBLink: 'https://tidepool-project.helpscoutdocs.com/article/6-how-to-install-or-upgrade-the-tidepool-uploader-gen',
```

Comment:
If so, we need to set this up. Are you volunteering to spend time wrangling whatever needs to be done (probably mucking around with the WP site) to do that now? We need to get this out as fast as possible, no matter what kind of link is in it (as there is no link at all currently), and we have yet to decide what level of testing process we're comfortable with.